<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventoryWindow</class>
 <widget class="QMainWindow" name="InventoryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Система керування складом</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #1E1E2E;
    color: #E0E0E0;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 11pt;
}

QWidget#centralwidget {
    background-color: #1E1E2E;
    border-radius: 10px;
}

QLabel {
    color: #E0E0E0;
    font-weight: bold;
}

QLineEdit {
    background-color: #2A2A3A;
    color: #E0E0E0;
    border: 1px solid #3A3A4A;
    border-radius: 6px;
    padding: 6px 10px;
    selection-background-color: #5A5A6A;
}
QLineEdit:focus {
    border: 1px solid #6A6A7A;
    background-color: #323242;
}

QSpinBox, QDoubleSpinBox {
    background-color: #2A2A3A;
    color: #E0E0E0;
    border: 1px solid #3A3A4A;
    border-radius: 6px;
    padding: 6px 10px;
    selection-background-color: #5A5A6A;
}
QSpinBox:focus, QDoubleSpinBox:focus {
    border: 1px solid #6A6A7A;
    background-color: #323242;
}

QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                stop:0 #FF6B9D, stop:0.5 #C06C84, stop:1 #6C5B7B);
    color: #FFFFFF;
    border: none;
    border-radius: 8px;
    padding: 8px 20px;
    font-weight: bold;
    min-width: 80px;
}
QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                stop:0 #FF85AD, stop:0.5 #D07C94, stop:1 #7C6B8B);
}
QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                stop:0 #E5517D, stop:0.5 #A05C74, stop:1 #5C4B6B);
}

QTableWidget {
    background-color: #1E1E2E;
    color: #D0D0D0;
    border: 1px solid #3A3A4A;
    gridline-color: #3A3A4A;
    selection-background-color: #4A4A5A;
    selection-color: #FFFFFF;
    border-radius: 6px;
}
QTableWidget::item {
    background-color: #1E1E2E;
}

QHeaderView::section {
    background-color: #2A2A3A;
    color: #C0C0C0;
    font-weight: bold;
    border: 1px solid #3A3A4A;
    padding: 6px;
    text-align: center;
}
QHeaderView::section:hover {
    background-color: #323242;
}

QHeaderView {
    background-color: #1E1E2E;
}

QTableWidget QTableCornerButton::section {
    background-color: #2A2A3A;
    border: 1px solid #3A3A4A;
}

QTableWidget::item {
    color: #E0E0E0;
}

QScrollBar:vertical {
    background-color: #2A2A3A;
    width: 12px;
    margin: 0px;
    border-radius: 6px;
}
QScrollBar::handle:vertical {
    background-color: #4A4A5A;
    min-height: 20px;
    border-radius: 6px;
}
QScrollBar::handle:vertical:hover {
    background-color: #5A5A6A;
}
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    height: 0px;
}
QScrollBar:horizontal {
    background-color: #2A2A3A;
    height: 12px;
    border-radius: 6px;
}
QScrollBar::handle:horizontal {
    background-color: #4A4A5A;
    min-width: 20px;
    border-radius: 6px;
}
QScrollBar::handle:horizontal:hover {
    background-color: #5A5A6A;
}
QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal {
    width: 0px;
}

QSpinBox, QDoubleSpinBox {
    background-color: #2A2A3A;
    color: #E0E0E0;
    border: 1px solid #3A3A4A;
    border-radius: 6px;
    padding: 6px 10px;
    selection-background-color: #5A5A6A;
}
QSpinBox:focus, QDoubleSpinBox:focus {
    border: 1px solid #6A6A7A;
    background-color: #323242;
}
QSpinBox::up-button, QDoubleSpinBox::up-button {
    background-color: #3A3A4A;
    border: 1px solid #4A4A5A;
    border-radius: 3px;
    width: 18px;
    margin: 2px;
}
QSpinBox::up-button:hover, QDoubleSpinBox::up-button:hover {
    background-color: #4A4A5A;
}
QSpinBox::down-button, QDoubleSpinBox::down-button {
    background-color: #3A3A4A;
    border: 1px solid #4A4A5A;
    border-radius: 3px;
    width: 18px;
    margin: 2px;
}
QSpinBox::down-button:hover, QDoubleSpinBox::down-button:hover {
    background-color: #4A4A5A;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="tableLabel">
      <property name="text">
       <string>&lt;h3&gt;Список товарів на складі&lt;/h3&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item>
     <widget class="QLabel" name="addLabel">
      <property name="text">
       <string>&lt;h3&gt;Додати товар&lt;/h3&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="inputLayout">
      <item>
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Назва:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameLineEdit">
        <property name="placeholderText">
         <string>Введіть назву товару</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="priceLabel">
        <property name="text">
         <string>Ціна:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="priceSpinBox">
        <property name="suffix">
         <string> грн</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="quantityLabel">
        <property name="text">
         <string>Кількість:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="quantitySpinBox">
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Додати</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="searchLayout">
      <item>
       <widget class="QLabel" name="searchLabel">
        <property name="text">
         <string>Знайти за ціною:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="searchPriceSpinBox">
        <property name="suffix">
         <string> грн</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>Пошук</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showAllButton">
        <property name="text">
         <string>Показати всі</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
